udata/frontend/markdown.py

```diff
@@ -86,9 +86,14 @@ def __call__(self, stream, source_tooltip=False, wrap=True):
         if source_tooltip:
             callbacks.append(source_tooltip_callback)
         
+        allowed_attr_md = current_app.config['MD_ALLOWED_ATTRIBUTES']
+        
+        if current_app.config['MD_ALLOWED_IMG']:
```

Comment:
what about smtg like this?
```suggestion
        if 'img' in current_app.config['MD_ALLOWED_ATTRIBUTES']:
```

Reply:
Good idea it seems cleaner but maybe less understandable from the config file. But i like the look of it 👍

Reply:
The purpose here was to use a virgin variable to avoid modifying the config dict one.
if I use this:
`if 'img' in current_app.config['MD_ALLOWED_ATTRIBUTES']:`
I can't do something like this after:
`current_app.config['MD_ALLOWED_ATTRIBUTES']['img'].append('src')`
Because we end up with something like this after a few reloads:
`{'a': ['href', 'title', 'rel', 'data-tooltip'], 'abbr': ['title'], 'acronym': ['title'], 'img': ['src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src', 'src']}`